dietpi/dietpi-software

```diff
@@ -8385,6 +8416,223 @@ _EOF_
 			[[ -f '/etc/exports.d/dietpi.exports' ]] || echo '/mnt/dietpi_userdata *(rw,async,no_root_squash,fsid=0,crossmnt,no_subtree_check)' > /etc/exports.d/dietpi.exports
 
 		fi 
+
+		software_id=184 # Tor Relay
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
+
+			local nickname=
+
+			G_WHIP_BUTTON_CANCEL_TEXT='Skip'
+
+			if G_WHIP_INPUTBOX "Pick a nickname for your relay. (This is optional)"; then
+
+				nickname=$G_WHIP_RETURNED_VALUE
+
+			fi
+
+			local email=
+
+			G_WHIP_BUTTON_CANCEL_TEXT='Skip'
+
+			if G_WHIP_INPUTBOX "Pick a contact email for your relay. (This is optional but encouraged, and it will be published depending on your relay)"; then
+
+				email=$G_WHIP_RETURNED_VALUE
+
+			fi
+
+			local orport=443
+			local invalid_text=
+			while :
+			do
+
+				G_WHIP_DEFAULT_ITEM=$orport
+				if G_WHIP_INPUTBOX "${invalid_text}Please enter the ORPort that people will use to connect to your Tor relay:\n
+NB: This port needs to be forwarded by your router and/or opened in your firewall settings. Default value is: 443. Do not use default value if running a webserver\n.
+Do not use 9001, as this port is commonly associated with Tor and censors might scan for it." && disable_error=1 G_CHECK_VALIDINT "$G_WHIP_RETURNED_VALUE" 0; then
+					orport=$G_WHIP_RETURNED_VALUE
+					break
+				else
+					invalid_text='[ERROR] No valid entry found, value needs to be a sequence of integers. Please retry...\n\n'
+				fi
+
+			done
+
+			local ipv6=
+
+			G_WHIP_BUTTON_CANCEL_TEXT='Skip'
+
+			if ! grep -q 'CONFIG_ENABLE_IPv6=1' /boot/dietpi.txt; then
```

Comment:
The queck is done the opposite way, isn't it? Only ask to enter the IPv6 address if IPv6 is enabled:
```
if grep -q '^[[:blank:]]*CONFIG_ENABLE_IPv6=1' /boot/dietpi.txt; then
```
Leading `^[[:blank:]]*` just to be sure the line is not commented.

Btw, while most users will have IPv6 enabled, most won't have a static IPv6 address. Not sure how it works, if a domain name would be an alternative, but probably we could add the suggestion to leave it blank as well when no static public IPv6 address is available.